SX1272/SX1272_LoRaRadio.h

```diff
@@ -367,7 +374,7 @@ class SX1272_LoRaRadio: public LoRaRadio {
     // If the chip fails to transmit, its a fatal error, reflecting
     // some catastrophic bus failure etc. We wish to have the control
     // back from the driver in such a case.
-    mbed::Timeout tx_timeout_timer;
+    ALIAS_LORAWAN_TIMER tx_timeout_timer;
```

Comment:
Why here? Should be defined in base class

Reply:
Please check https://github.com/ARMmbed/mbed-semtech-lora-rf-drivers/pull/31#discussion_r224045203